awx/main/models/inventory.py

```diff
@@ -803,6 +803,14 @@ def _get_related_jobs(self):
         return UnifiedJob.objects.non_polymorphic().filter(Q(job__inventory=self.inventory) | Q(inventoryupdate__inventory_source__groups=self))
 
 
+class HostMetric(models.Model):
+    hostname = models.CharField(primary_key=True, max_length=512)
+    first_automation = models.DateTimeField(
+        auto_now_add=True, null=False, db_index=True, help_text=_('DateTime ( with timestamp ) , when the object was first created')
```

Comment:
`When the object was first created` -> `When the host was first automated against.`
`When the object was last modified` -> `When the host was last automated against.`

Reply:
Out of my feedback, this is probably the one item I feel strongly about addressing before merging.

Reply:
I have made the required changes